services/export/shared/transactionsExport.js

```diff
@@ -0,0 +1,462 @@
+ *
+ */
+const Moment = require('moment');
+const MomentRange = require('moment-range');
+const excelJS = require('exceljs');
+const xlsx = require('node-xlsx');
+
+const moment = MomentRange.extendMoment(Moment);
+
+const {
+	CacheRedis,
+	Exceptions: {
+		NotFoundException,
+		ValidationException,
+	},
+	Queue,
+} = require('lisk-service-framework');
+
+const {
+	getLisk32AddressFromPublicKey,
+	getLegacyAddress,
+} = require('./helpers/account');
+
+const { getCurrentChainID, resolveReceivingChainID } = require('./helpers/chain');
+const { MODULE, COMMAND, EVENT, MODULE_SUB_STORE } = require('./helpers/constants');
+
+const {
+	requestIndexer,
+	requestConnector,
+	requestAppRegistry,
+} = require('./helpers/request');
+
+const {
+	getDaysInMilliseconds,
+	dateFromTimestamp,
+	timeFromTimestamp,
+} = require('./helpers/time');
+
+const {
+	normalizeTransactionAmount,
+	normalizeTransactionFee,
+	checkIfSelfTokenTransfer,
+} = require('./helpers/transaction');
+
+const config = require('../config');
+const fields = require('./excelFieldMappings');
+
+const requestAll = require('./requestAll');
+const FilesystemCache = require('./csvCache');
+const regex = require('./regex');
+
+const partials = FilesystemCache(config.cache.partials);
+const staticFiles = FilesystemCache(config.cache.exports);
+
+const noTransactionsCache = CacheRedis('noTransactions', config.endpoints.volatileRedis);
+
+const DATE_FORMAT = config.excel.dateFormat;
+const TIME_FORMAT = config.excel.timeFormat;
+const MAX_NUM_TRANSACTIONS = 10000;
+
+let tokenModuleData;
+let legacyModuleData;
+
+const getTransactions = async (params) => requestIndexer('transactions', params);
+
+const getGenesisBlock = async () => requestIndexer(
+	'blocks',
+	{
+		limit: 1,
+		sort: 'height:asc',
+	},
+);
+
+const getAddressFromParams = (params) => params.address
+	|| getLisk32AddressFromPublicKey(params.publicKey);
+
+const getTransactionsInAsc = async (params) => getTransactions({
+	address: getAddressFromParams(params),
+	sort: 'timestamp:asc',
+	timestamp: params.timestamp,
+	limit: params.limit || 10,
+	offset: params.offset || 0,
+});
+
+const validateIfAccountExists = async (address) => {
+	const { data: tokenBalances } = await requestIndexer('token.balances', { address });
+	return !!tokenBalances.length;
+};
+
+const getCrossChainTransferTransactionInfo = async (params) => {
+	const allEvents = await requestIndexer('events', {
+		topic: params.address,
+		timestamp: params.timestamp,
+		module: MODULE.TOKEN,
+		name: EVENT.CCM_TRANSFER,
+		sort: 'timestamp:desc',
+	});
+
+	const transactions = [];
+	const ccmTransferEvents = allEvents.data
+		.filter(event => event.data.recipientAddress === params.address);
+
+	for (let i = 0; i < ccmTransferEvents.length; i++) {
+		const [transactionID] = ccmTransferEvents[i].topics;
+
+		transactions.push({
+			id: transactionID,
+			moduleCommand: `${MODULE.TOKEN}:${COMMAND.TRANSFER_CROSS_CHAIN}`,
```

Comment:
Shouldn't this be `interoperability:submitSidechainCrossChainUpdate` or `interoperability:submitMainchainCrossChainUpdate` instead?